examples/js/utils/RoughnessMipmapper.js

```diff
@@ -0,0 +1,174 @@
+/**
```

Comment:
For new modules, we're adding files to `examples/jsm`, and omitting them from `examples/js` (and the modularize script) entirely.

Reply:
Ah, that's news to me. Should I change it?

Reply:
Removed.